lms/views/api/blackboard/sync.py

```diff
@@ -0,0 +1,90 @@
+from lms.security import Permissions
+from lms.services import CanvasAPIError
+from pyramid.view import view_config
+from lms.views import (
+    CanvasGroupSetEmpty,
```

Comment:
These all say `Canvas` but I think we probably wan't to remove that part and reuse them.

Reply:
:+1: